lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
+    grid-template-rows: 39px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__top-row {
+    -ms-grid-row-align: start;
+    align-self: start;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    grid-area: top-row;
+    padding-top: $nx-spacing-xs;
+  }
+
+  .nx-product__bottom-row {
```

Comment:
This element is getting a height based on its font-size - which comes across a little weird because all of its children have a _different_ (smaller) font size.  So it height and the visible distance between the text here and the text above ultimately comes down to the strut, which I'm not super comfortable with.

Reply:
> This element is getting a height based on its font-size - which comes across a little weird because all of its children have a _different_ (smaller) font size. So the height of this element and the visible distance between the text here and the text above ultimately comes down to the strut, which I'm not super comfortable with.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/fb3ae07c988230b25e10d6b58146a8e517236ec0